naga/src/front/wgsl/lower/mod.rs

```diff
@@ -3026,13 +3026,14 @@ impl<'source, 'temp> Lowerer<'source, 'temp> {
         );
 
         for (arg_index, &arg) in arguments.iter().enumerate() {
-            let ty = ctx.typifier()[arg].inner_with(&ctx.module.types);
+            let arg_type_resolution = &ctx.typifier()[arg];
+            let arg_inner = arg_type_resolution.inner_with(&ctx.module.types);
             log::debug!(
-                "Supplying argument {arg_index} of type {}",
-                crate::common::DiagnosticDisplay((ty, ctx.module.to_ctx()))
+                "Supplying argument {arg_index} of type {:?}",
+                crate::common::DiagnosticDebug((arg_type_resolution, &ctx.module.types))
```

Comment:
thought: What, no `for_debug` here? 👀

Reply:
I changed this to use `ForDebugWithTypes`. Thanks.